library/general/noupdateserviceaccount/suite.yaml

```diff
@@ -0,0 +1,20 @@
+kind: Suite
+apiVersion: test.gatekeeper.sh/v1alpha1
+metadata:
+  name: noupdateserviceaccount
+tests:
+  - name: noupdateserviceaccount
+    template: template.yaml
+    constraint: samples/noupdateserviceaccount/constraint.yaml
+    cases:
+      - name: example-allowed
+        object: samples/noupdateserviceaccount/example_allowed.yaml
+        assertions:
+          - violations: no
+      # Since the tests are currently stateless, they can't exercise level-triggered
+      # events like changing a field. For now, manually test example_disallowed.yaml
```

Comment:
@willbeason Would this type of policy that requires `input.review.oldObject`  need to be tested with `gator test`?

Reply:
I think yes

Reply:
Yeah - we're considering allowing test authors to optionally specify a full AdmissionReview instead of an object so that they can specify all of the other fields

Reply:
Do I need to add something to the PR for that?

Reply:
Nope - this is a side design discussion for something completely different. No action necessary on your part!

Reply:
Filed https://github.com/open-policy-agent/gatekeeper/issues/1843